ResourceManager/GitExtensionsFormBase.cs

```diff
@@ -26,7 +26,8 @@ public GitExtensionsFormBase()
         {
             _initialiser = new GitExtensionsControlInitialiser(this);
 
-            ShowInTaskbar = Application.OpenForms.Count <= 0;
+            var openFormCount = Application.OpenForms.Count;
+            ShowInTaskbar = openFormCount <= 0;
```

Comment:
Can we show all forms in the task bar? Modern versions of Windows (7, 10) will group these together. I wonder if keeping such forms visible in the taskbar might address some of the FormCommit glitches I'd previously assumed were related to it being modal.

Another workaround might be to use P/Invoke to clear the `WS_EX_APPWINDOW` extended style flag from the form. From memory you then have to hide and re-show the window for it to take effect.

I can't see anything wrong with the fix proposed by this PR. Just wondering if we can avoid it altogether by showing everything in the taskbar.